Reword Payment Flow Implementation section (#660)
centrapay/centrapay-docs: src/content/guides/farmlands-pos-integration.mdx

## Payment Flow Implementation

> The Payment Flow can only be used by existing Farmlands Card Partners. Card Partners must complete the following steps to accept Farmlands transactions. 

Reviewer: There is an extra space at the end of the line

1. Each Card Partner site must be correctly set up as a [Merchant](https://docs.centrapay.com/api/merchants) in Centrapay’s system.
2. Centrapay needs to supply Card Partners with API keys to allow them to authenticate their API requests.
3. Card Partners must meet [basic payment requirements](#basic-requirements), including supporting [Payment Conditions](#payment-conditions) and [Refunds](#refunds) in order to implement the Barcode Payment Flow.

> 4. Card Partners should extend the Barcode payment Flow to meet any additional requirements, these optional extensions include:

Reviewer: We start with Card Partners twice maybe change to something like the following

```suggestion
4. For additional requirements, Card Partners should extend the Barcode payment flow with these optional extensions:
```

Reviewer 2: I feel that just adds unnecessary complexity to the sentence. Also we have got to merge this by the end of today. I'm going to resolve this comment.

    - [Pre Auth payments](#pre-auth) when there is no invoice number available at the point of sale, or the purchase cannot be fulfilled immediately

>     - [Cardholder not physically present](#cardholder-not-present)

Reviewer 3: - This bullet feels a little incomplete - any reason why we didn't use the original content?

```suggestion
    - Supporting payments when the [Cardholder is not physically present](#cardholder-not-present).
```
